README.md

```diff
@@ -1,6 +1,6 @@
 ## coreJSON Library
 
-This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
+This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and is suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
```

Comment:
Can we change the language? "This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for constrained embedded devices. It is distributed under the [MIT open source license](LICENSE)

Reply:
(this has already been made, tanmoy was viewing the old file)